proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,113 @@
+
+==============
+OCDEP: Division Identifier Governance
+==============
+
+:Created: 2017-10-26
+:Author: Donny Bridges
+:Status: Proposed
+
+Overview
+========
+
+A formalized governance model for the creation, acceptance, and upkeep of Open Civic Data Division Identifiers (OCDIDs).
+
+Rationale
+=========
+
+Recently, a number of civic data organizations, tech companies, and US state governments have expressed interest in taking a greater role in the creation, upkeep, and wider adoption of OCDIDs. Currently, the governance structure for division IDs  as defined in OCDEP2 is "informal... led by the project’s early contributors and informed by the Open Civic Data Google Group." Increased adoption and interest makes this informality untenable for reasons including:
+
+* Bottlenecks arising when large amounts of new identifiers are proposed but only a few people have permission to review and approve pull requests.
+* Subject-matter experts and organizations that have extensive experience working with OCDIDs are willing to take greater responsibility, but currently do not have a formal path for doing so.
+* Government entities and other local experts who may wish to contribute to the OCDID repository do not have a clear idea of how the process for contirbution works, making it more difficult for them to justify contributing.
+
+We propose to formalize this structure in order to relieve some of the burden on those currently responsible for maintaining the repository, as well as allowing those whose work depends on OCDIDs to have more direct control on their upkeep.
+
+
+Implementation
+==============
+
+Roles & Responsibilities
+------------------------
+``User``
+~~~~~~~~
+Anyone using OCDIDs may contribute through Slack/Google group discussion, reporting issues, or assisting with subject-matter expertise.
+These channels for communication will be publicaly displayed on the Open Civic Data website.
+
+``Contributor``
+~~~~~~~~~~~~~~~
+Any individual or organization ``user`` may become a ``contributor`` by submitting a pull request adding, correcting, or aliasing jurisdictional OCDIDs.
+First time ``contributors`` will be advised to consult with an existing ``committer`` before submitting their first PR.
+
+``Committer`` 
+~~~~~~~~~~~~~
+``Committers`` have the ability to approve and commit pull requests from ``contributors`` and other ``committers``. 
+An initial cohort of ``committers`` will be determined by current project contributors.
+Any individual or organization who is a ``contributor`` and who agrees to the repsonsibilities listed here may apply to become a ``committer``. 
+
+* There will be a two month period where existing ``committers`` may raise objections to an applicant, otherwise the applying organization/individual will be considered approved.
+* In the case that an objection is raised, a discussion and vote will be had amongst existing ``committers``, with a simple majority necessary for approval.
```

Comment:
Need some definition of a quorum probably.

Reply:
Sure